native/src/main/java/io/ballerina/persist/nodegenerator/CreateSyntaxTree.java

```diff
@@ -0,0 +1,204 @@
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package io.ballerina.persist.nodegenerator;
+
+import io.ballerina.toml.syntax.tree.AbstractNodeFactory;
+import io.ballerina.toml.syntax.tree.DocumentMemberDeclarationNode;
+import io.ballerina.toml.syntax.tree.DocumentNode;
+import io.ballerina.toml.syntax.tree.KeyNode;
+import io.ballerina.toml.syntax.tree.KeyValueNode;
+import io.ballerina.toml.syntax.tree.NodeFactory;
+import io.ballerina.toml.syntax.tree.NodeList;
+import io.ballerina.toml.syntax.tree.SyntaxTree;
+import io.ballerina.toml.syntax.tree.TableArrayNode;
+import io.ballerina.toml.syntax.tree.TableNode;
+import io.ballerina.toml.syntax.tree.Token;
+import io.ballerina.toml.validator.SampleNodeGenerator;
+import io.ballerina.tools.text.TextDocument;
+import io.ballerina.tools.text.TextDocuments;
+
+import java.io.IOException;
+import java.io.PrintStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+
+import static io.ballerina.persist.PersistToolsConstants.DEFAULT_DATABASE;
+import static io.ballerina.persist.PersistToolsConstants.DEFAULT_HOST;
+import static io.ballerina.persist.PersistToolsConstants.DEFAULT_PASSWORD;
+import static io.ballerina.persist.PersistToolsConstants.DEFAULT_PORT;
+import static io.ballerina.persist.PersistToolsConstants.DEFAULT_USER;
+import static io.ballerina.persist.PersistToolsConstants.KEY_DATABASE;
+import static io.ballerina.persist.PersistToolsConstants.KEY_HOST;
+import static io.ballerina.persist.PersistToolsConstants.KEY_PASSWORD;
+import static io.ballerina.persist.PersistToolsConstants.KEY_PORT;
+import static io.ballerina.persist.PersistToolsConstants.KEY_USER;
+
+
+/**
+ * Class to create syntax tree for Config.toml.
+ */
+public class CreateSyntaxTree {
+    private static final PrintStream outStream = System.out;
+    private static String[] nodeMap = {KEY_HOST, KEY_PORT, KEY_USER, KEY_DATABASE, KEY_PASSWORD};
+    private static String[] defaultValues = {DEFAULT_HOST, DEFAULT_PORT, DEFAULT_USER, DEFAULT_DATABASE,
+            DEFAULT_DATABASE};
+
+    /**
+     * Method to create a new Config.toml file with database configurations.
+     */
+    public static SyntaxTree createToml() {
+        NodeList<DocumentMemberDeclarationNode> moduleMembers = AbstractNodeFactory.createEmptyNodeList();
+        moduleMembers = populateNodeList(moduleMembers, true);
+        Token eofToken = AbstractNodeFactory.createIdentifierToken("");
+        DocumentNode documentNode = NodeFactory.createDocumentNode(moduleMembers, eofToken);
+        TextDocument textDocument = TextDocuments.from(documentNode.toSourceCode());
+        SyntaxTree syntaxTree = SyntaxTree.from(textDocument);
+        return syntaxTree;
```

Comment:
```suggestion
        return SyntaxTree.from(textDocument);
```

Reply:
Removed in [e4c809d](https://github.com/ballerina-platform/persist-tools/pull/9/commits/e4c809da564408437e379bbe491d495ca2cbd2e8)